config/google_cloud_exporter/iis/config.yaml

```diff
@@ -0,0 +1,30 @@
+receivers:
+  iis:
+    collection_interval: 10s
```

Comment:
Lets be consistent with 60s collection interval. I believe the Google minimum is 30 seconds.